jpa/eclipselink.jpa.testapps/jpa.test.ddlgeneration/src/test/java/org/eclipse/persistence/testing/tests/jpa/ddlgeneration/FractionalSecondsPrecisionTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ * Copyright (c) 2025 IBM Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.persistence.testing.tests.jpa.ddlgeneration;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.EntityTransaction;
+import junit.framework.Test;
+import junit.framework.TestSuite;
+import org.eclipse.persistence.internal.databaseaccess.DatabasePlatform;
+import org.eclipse.persistence.internal.databaseaccess.Platform;
+import org.eclipse.persistence.testing.framework.jpa.junit.JUnitTestCase;
+import org.eclipse.persistence.testing.models.jpa.ddlgeneration.Event;
+
+public class FractionalSecondsPrecisionTest extends JUnitTestCase {
+
+    /** Default persistence unit name. */
+    private static final String PU_NAME = "fractional";
+
+    /**
+     * List of {@code Event}s. Array index matches ID.
+     */
+    private static final Event[] EVENTS = new Event[] {
+            null, // Skip index 0
+            new Event(1L,
+                      LocalDateTime.of(2025, 6, 11, 12, 0, 0, 123_451_234)),
+    };
+
+    public FractionalSecondsPrecisionTest() {
+        super();
+    }
+
+    public FractionalSecondsPrecisionTest(String name) {
+        super(name);
+    }
+
+    public static Test suite() {
+        TestSuite suite = new TestSuite("Null Binding DateTime");
+        suite.addTest(new FractionalSecondsPrecisionTest("testSetup"));
```

Comment:
Unable to test all, Missing `suite.addTest(new FractionalSecondsPrecisionTest("testLocalDateTimeCustomPrecision"));`

Reply:
This is still work in progress. You are too fast. :D